COMMITS.md

```diff
@@ -16,10 +16,18 @@ Use this git hook to auto-check your commit messages. Save the following snippet
 
 ```
 #!/bin/sh
+
+commit_msg=$(cat "$1")
+if echo "$commit_msg" | grep -qE "^fixup! "; then
+  # Skip validation in case of fixup commit
+  exit 0
+fi
+
 if ! grep -qE "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|revert)(\((blockchain-link|components|components-storybook|integration-tests|suite|suite-data|suite-desktop|suite-native|suite-onboarding|suite-storage|suite-web)\))?: " "$1" ; then
```

Comment:
uh, this seems to be wildly outdated as well

Reply:
So what are the current rules?

Reply:
just that there are way more packages than this. also having this hardcoded is hard to mantain. I wouldn't mind simply removing pakage names rules from here

Reply:
we can delete the recommendation of the hook completely, or at least remove the scope part check...